### PR TITLE
feat: Add Glue Workflows resource

### LINKED ISSUE #1302: AWS::Glue::Workflow resource
Implementation of AWS::Glue::Workflow resource.

### PR DESCRIPTION
**Pull request by irmatov**
Closes #1302

### CODE REVIEW

File: resources/services/glue/workflows.go [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+	result, err := svc.GetTags(ctx, &glue.GetTagsInput{
+		ResourceArn: aws.String(workflowARN(cl, aws.ToString(resource.Item.(types.Workflow).Name))),
+	})
+	if err != nil {
+		if cl.IsNotFoundError(err) {
+			return nil
+		}
+		return diag.WrapError(err)
+	}
+	return diag.WrapError(resource.Set(c.Name, result.Tags))
```
Comment (candiduslynx) [inline resources/services/glue/workflows.go:252]: Use [`client.TagsToMap`](https://github.com/cloudquery/cq-provider-aws/blob/main/client/helpers.go#L415)

**Review by candiduslynx — CHANGES_REQUESTED** [review]

File: resources/services/glue/workflows.go [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+			},
+			{
+				Name:          "last_run_starting_event_batch_condition_batch_size",
+				Description:   "Number of events in the batch.",
+				Type:          schema.TypeBigInt,
+				Resolver:      schema.PathResolver("LastRun.StartingEventBatchCondition.BatchSize"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "last_run_starting_event_batch_condition_batch_window",
```
Comment (amanenk) [inline resources/services/glue/workflows.go:127]: Should we avoid repeating batch in this and other columns?

**Review by amanenk — APPROVED** [review]
LGTM. a few suggestions

File: resources/services/glue/workflows.go (hunk 1 above)
Comment (irmatov) [reply resources/services/glue/workflows.go:252]: it's already a map :)

**Review by candiduslynx — APPROVED** [review]

File: resources/services/glue/workflows.go (hunk 2 above)
Comment (irmatov) [reply resources/services/glue/workflows.go]: Adjusted as suggested, thanks!

**Pull request merged**